BSDmakefile

```diff
@@ -0,0 +1,5 @@
+default:
```

Comment:
Another way to do it would be to name our Makefile GNUmakefile but I'm nervous about that not working on some version of GNU Make. But that is probably an unfounded fear.

I had hoped that Solaris type systems would also work with this BSDmakefile solution but that doesn't seem to be the case. So maybe we have to document gmake anyway...